### ISSUE TITLE
The dist/xterms.css file is missing from version 3.14.0

### ISSUE THREAD

**Issue by domgoer** [xtermjs/xterm.js#2150]
## Details
- Browser and browser version: ----
- OS version: ---
- xterm.js version: 3.14.0

```
GET file:///Users/dmc/test-terminal/node_modules/xterm/dist/xterm.css net::ERR_FILE_NOT_FOUND
```

**Comment by ixrock** [comment]
@domgoer 
you still can include it with: `@import "~xterm/src/xterm.css";` (webpack)

**Comment by Tyriar** [comment]
You're right it's missing, luckily this should be the last time something like this happens as that whole folder is going away in v4

**Comment by Tyriar** [comment]
Fixed in https://github.com/xtermjs/xterm.js/commit/13133d699f73952e8518a419c0e4ea0696be2f0a

**Comment by Tyriar** [comment]
Released 3.14.1
